build.sc

```diff
@@ -233,12 +233,14 @@ object `llvm-firtool` extends JavaModule with ChipsAlliancePublishModule {
 // ******************** WARNING ********************
 // This is extremely manual and changing dependencies IN ANY WAY (including bumping version)
 // requires carefully checking the packages to shade and dynamic ivy deps in the outer project
-object `firtool-resolver` extends Cross[FirtoolResolver]("2.13", "2.12")
-trait FirtoolResolver extends CrossScalaModule with ChipsAlliancePublishModule { root =>
+object `firtool-resolver` extends Cross[FirtoolResolver]("2.13", "2.12", "3.4", "3.3")
+trait FirtoolResolver extends CrossSbtModule with ChipsAlliancePublishModule { root =>
```

Comment:
Rather than switching this to SBT style, I would rather stick with vanilla Mill style. We only use SBT style in Chisel for historical reasons, we could just override `millSourcePath` here. Fortunately, we don't even need to do that as I will explain below.